docs/source/testing/index.rst

```diff
@@ -3,35 +3,31 @@
 Testing OpenFAST
 ================
 
-The OpenFAST test suite consists of glue code and module level regression tests
-and unit tests. The regression tests compare locally generated solutions to
-a set of baseline solutions. The unit tests ensure that individual subroutines
-are functioning as intended.
+OpenFAST is a complex software with many moving parts. In order to
+maintain stability in new and existing code, a test suite is included
+directly in the source code. Two primary levels of tests exist:
+regression tests at the highest level and unit tests at the lowest
+level. The regression tests compare locally generated results with
+stored "baseline" results. This test gives an indication of whether the
```

Comment:
Change "This test gives" to "These tests give".

Reply:
Good catch, changed